src/nnet3/nnet-optimize.h

```diff
@@ -44,7 +47,8 @@ struct NnetOptimizeOptions {
                          remove_assignments(true),
                          initialize_undefined(true),
                          move_sizing_commands(true),
-                         allocate_from_other(true) { }
+                         allocate_from_other(true),
+                         num_computations_cached(20) { }
```

Comment:
Sorry, now that I see this code I realize that I told you something wrong.  I though this was a class for the CachingOptimizingCompiler's options, but it was not.  Sorry.  You can revert that part of the code to the way it was, I think.